src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue5669.cs

```diff
@@ -25,8 +26,24 @@ public void SearchBarMaxLength()
 			// This causes the image generated in CI to differ from local runs.
 			Assert.That(App.WaitForElement("Sear").GetText(), Is.EqualTo("Sear"));
 #else
-			VerifyScreenshot();
+			//VerifyScreenshot();
 #endif
 		}
+
+		[Test]
+		[Category(UITestCategories.SearchBar)]
+		[Ignore("The necessary Azure OpenAI parameters are still to be added on CI")]
+		public async Task SearchBarMaxLengthWithAI()
+		{
+			await App.ExecuteWithAI(@"
```

Comment:
Allow using human language to define the steps to perform in the test.
![ai-test-steps](https://github.com/user-attachments/assets/78613057-e395-4c9c-bcdc-23e1c3d9a704)